tests/suites/deploy/deploy_bundles.sh

```diff
@@ -35,8 +35,8 @@ run_deploy_cmr_bundle() {
 
 	ensure "test-cmr-bundles-deploy" "${file}"
 
-	# mysql charm does not have stable channel, so we use edge channel
-	juju deploy mysql --channel=edge
+	# mysql charm does not have stable channel, so we use candidate channel
```

Comment:
```suggestion
	# mysql charm does not have stable channel, so we use the candidate channel
```

Does this actually work? Looking at:
```
$ juju info mysql
...
channels: |
  latest/stable:     –
  latest/candidate:  –
  latest/beta:       –
  latest/edge:       71  2022-07-22  (71)  2MB
```

That says they aren't even supported to have a `candidate` channel. So juju should fail to deploy mysql from anything other than edge (as of today).
I'd actually rather we move away from `mysql` until data platform has really gone to stable with it.

Reply:
We've started to use postgresql elsewhere for this reason. We should be consistent.

Reply:
`mysql` ("candidate") worked in that case: https://pastebin.canonical.com/p/yV6v52bYjj/
Moreover, "edge" does not work because "edge" does not have `db` relation.

I agree that we need to minimize the usage of MySQL, but in this test, we use it in pair with `wordpress`. So, if change `mysql`, we need to change `wordpress`, too. An alternative could be pair: `postgresql`, `grafana`.